feature-flags/feature-flags.interface.ts

```diff
@@ -27,4 +27,6 @@ export interface FeatureFlagsInterface {
   FEATURE_TEAM_PROJECTS: boolean; // disable Team Projects page and related features
   FEATURE_ECOSYSTEM_FINANCES_DASHBOARD_PAGE: boolean; // disable Accounts Finances Page
   FEATURE_ROADMAP_MILESTONES: boolean; // disable roadmap milestones page
+  FEATURE_ENDGAME_MILESTONES_PHASE_1: boolean; // disable Milestone Phase 1 in Endgame page
```

Comment:
This feature flag is not used, it was renamed to `FEATURE_ROADMAP_MILESTONES` early so this should be deleted